tests/patterns/kotlin/single_interpolated_ident.sgrep

```diff
@@ -0,0 +1 @@
+"${FOO}
```

Comment:
I'm surprised this pattern works without a close quote

Reply:
fck

Reply:
i guess it was a tree-sitter partial parse. good catch though!